### ISSUE TITLE
Increase code coverage

### ISSUE THREAD

**Issue by omaciel** [omaciel/pytest-fauxfactory#10]
```shell
coverage report -m
Name                    Stmts   Miss  Cover   Missing
-----------------------------------------------------
pytest_fauxfactory.py      68      2    97%   24, 39
```

**Comment by ldjebran** [comment]
now showing 
```bash
coverage report -m
Name                    Stmts   Miss  Cover   Missing
-----------------------------------------------------
constants.py                2      0   100%
handlers.py                40      0   100%
helpers.py                  9      0   100%
marks.py                   18      1    94%   13
pytest_fauxfactory.py      11      0   100%
-----------------------------------------------------
TOTAL                      80      1    99%

```

**Comment by omaciel** [comment]
Fixed by 901a55b
